picard/ui/forms/ui_options_renaming.py

```diff
@@ -156,7 +156,7 @@ def retranslateUi(self, RenamingOptionsPage):
         self.move_files.setTitle(_("Move files when saving"))
         self.label.setText(_("Destination directory:"))
         self.move_files_to_browse.setText(_("Browse…"))
-        self.move_additional_files.setText(_("Move additional files (case insensitive):"))
+        self.move_additional_files.setText(_("Move additional files and/or folders (case insensitive):"))
```

Comment:
I'm wonering if "and/or" shouldn't be changed to just "and".

Reply:
Also, it would be more appropriate to use the term "subdirectories" rather than "folders", because folders could imply directories that are outside of the directory containing the file being processed.

Reply:
"subfolders" can work too? (shorter in English)

Reply:
Sure.  I remember the days when "folders" was a Windows thing and everyone else referred to them as "directories".  I guess now the two are synonymous, so I suppose "subfolders" is okay.  😜